userland/examples/nrf51_temp_sensor/main.c

```diff
@@ -1,8 +1,7 @@
+#include "temperature.h"
 #include <stdio.h>
 #include <timer.h>
```

Comment:
idk, uncrustify changed it or something somehow, but it should be ""---it's a local header file.